core/src/main/java/com/linecorp/armeria/common/AbstractHttpRequestBuilder.java

```diff
@@ -217,7 +217,7 @@ public AbstractHttpRequestBuilder queryParam(String name, Object value) {
         if (queryParams == null) {
             queryParams = QueryParams.builder();
         }
-        queryParams.setObject(name, value);
+        queryParams.addObject(name, value);
```

Comment:
Wondering if we should also modify `HttpRequestSetters#queryParams` for consistency with `HttpRequestSetters#headers`.
What do you think @ikhoon ?

Reply:
Consistency is important for users' experience. I prefer to modify them together.

Reply:
Sounds good to me.